app/assets/stylesheets/trips.css.scss

```diff
@@ -194,6 +194,40 @@
   margin: 15px 15px 15px 15px
 }
 
+.the-zambezi-river-trip-img {
+  height: 550px;
+  overflow: hidden;
+  position: relative;
+  background: image-url('zambezi.jpg'), no-repeat;
+  background-position: 0 -150px;
+  background-size: cover;
+  border: solid 3px #363636;
+  border-radius: 5px 5px 5px 5px;
+  margin: 15px 15px 15px 15px
+}
+
+.mexican-scuba-diving-trip-img {
+  height: 550px;
+  overflow: hidden;
+  position: relative;
+  background: image-url('cancun.jpg'), no-repeat;
+  background-size: cover;
+  border: solid 3px #363636;
+  border-radius: 5px 5px 5px 5px;
+  margin: 15px 15px 15px 15px
```

Comment:
Shorthand form for property `margin` should be written more concisely as `15px` instead of `15px 15px 15px 15px`<br>Declaration should be terminated by a semicolon